CRM/CiviGeometry/BAO/Geometry.php

```diff
@@ -333,11 +323,44 @@ public static function generateBounds($geometryID) {
     return ['left_bound' => $leftBound, 'bottom_bound' => $bottomBound, 'top_bound' => $topBound, 'right_bound' => $rightBound];
   }
 
-  public static function safeLoadAutoloader() {
-    $path = E::path('vendor/autoload.php');
-    if (file_exists($path)) {
-      require_once $path;
+  /**
+   * Convert Wkt Geoemtry to KML
+   * @param string $wkt
+   * @return string KML Geoemtry
+   */
+  public static function wkt2kml($wkt){
+    // Change coordinate format
+    $wkt = preg_replace("/([0-9\.\-]+) ([0-9\.\-]+),*/", "$1,$2 ", $wkt);
```

Comment:
I think it would be good to add a comment here referencing the blog post you found. Good for future readers to see where this came from.

Reply:
Done